### PR TITLE
fix: KeyboardEvent.keyCode is deprecated

### PR DESCRIPTION
**Pull request by kerlon5**
use KeyboardEvent.code instead
it is used to fix [camunda/camunda-modeler/2724](https://github.com/camunda/camunda-modeler/issues/2724)

I'm not sure, do you need an old check by KeyboardEvent.keyCode? Then we can use triple check

### CODE REVIEW

**Comment by kerlon5** [comment]
@barmac I can't add "needs review" label to PR

**Comment by barmac** [comment]
That is fine. We have an automation for this :)

**Comment by barmac** [comment]
It's OK, we don't need the flag to review the PR.

File: lib/features/keyboard/KeyboardUtil.js [hunk 1]
```diff
@@ -59,6 +59,10 @@ export function isPaste(event) {
   return isCmd(event) && isKey(KEYS_PASTE, event);
 }
 
+export function isCut(event) {
+  return isCmd(event) && isKey(KEYS_CUT, event);
+}
+
```
Comment (barmac) [inline lib/features/keyboard/KeyboardUtil.js:65]: Neither this nor `isSelectAll` are used anywhere in the code.

**Review by barmac — CHANGES_REQUESTED** [review]
While I believe the general direction of the PR, it does not replace keyCode with supported properties in the whole codebase. Notice that SearchPad is left intact: https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/search-pad/SearchPad.js#L94

File: lib/features/keyboard/KeyboardUtil.js (hunk 1 above)
Comment (kerlon5) [reply lib/features/keyboard/KeyboardUtil.js]: removed unsed code and keyCode on all codebase - ready to review

**Comment by barmac** [comment]
I will look into this again this week. Sorry for delay.

File: lib/features/dragging/Dragging.js [hunk 2]
```diff
@@ -282,7 +282,7 @@ export default function Dragging(eventBus, canvas, selection, elementRegistry) {
 
   function checkCancel(event) {
 
-    if (event.which === 27) {
+    if (event.code === 'Escape') {
```
Comment (barmac) [inline lib/features/dragging/Dragging.js:285]: This will break the feature for IE entirely: https://caniuse.com/keyboardevent-code

**Comment by nikku** [comment]
@barmac I believe we can ditch IE entirely now. New features we build (cf. https://github.com/bpmn-io/diagram-js/pull/687) don't have the legacy in sight, too.

**Review by barmac — APPROVED** [review]
I adjusted the PR and added a changelog update. Ready to merge.

**Comment by barmac** [comment]
So there is actually a polyfill for KeyboardEvent#key which I linked :)

**Pull request merged**